app/controllers/RemoveItemController.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions._
+import forms.RemoveItemFormProvider
+
+import javax.inject.Inject
+import models.{Mode, ReviewMode}
+import navigation.Navigator
+import pages.RemoveItemPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.UserAnswersService
+import views.html.RemoveItemView
+
+import scala.concurrent.Future
+
+class RemoveItemController @Inject()(
+                                       override val messagesApi: MessagesApi,
+                                       override val userAnswersService: UserAnswersService,
+                                       override val navigator: Navigator,
+                                       override val auth: AuthAction,
+                                       override val withMovement: MovementAction,
+                                       override val getData: DataRetrievalAction,
+                                       override val requireData: DataRequiredAction,
+                                       override val userAllowList: UserAllowListAction,
+                                       formProvider: RemoveItemFormProvider,
+                                       val controllerComponents: MessagesControllerComponents,
+                                       view: RemoveItemView
+                                     ) extends BaseNavigationController with AuthActionHelper {
+
+  def onPageLoad(ern: String, arc: String, idx: Int, mode: Mode): Action[AnyContent] =
+    authorisedDataRequestWithCachedMovementAsync(ern, arc) { implicit request =>
+      withAddedItemAsync(idx) {
+        _ =>
+          Future(Ok(view(
+            form = fillForm(RemoveItemPage(idx), formProvider()),
+            idx = idx,
+            call = routes.RemoveItemController.onSubmit(ern, arc, idx, mode)
+          )))
```

Comment:
Also, I don't think it would ever pre-fill this with a saved answer, because it's just a logic decision. It would be odd if it remembered the last answer and pre-loaded it in.